nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -236,11 +228,11 @@ def __init__(
             )
         else:
             assert attention_type == AttnType.cross_attn
-            self.query = tensor_parallel.ColumnParallelLinear(
+            self.query = tensor_parallel.ColumnLinear(
```

Comment:
You probably want to remove `tensor_parallel.` here.

Reply:
Done.